.jsbeautifyrc

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
why 4 spaces indent?

Reply:
Fixed.
P.S. I am always adding all team-members as reviewers it's just hard to push them to actually accept the invitation if you are not in the office (will be on Monday).